config/settings/base.py

```diff
@@ -367,3 +367,5 @@
 MAPBOX_TOKEN = env("MAPBOX_TOKEN", default=None)
 
 OPEN_EXCHANGE_RATES_API_ID = env("OPEN_EXCHANGE_RATES_API_ID", default=None)
+
+OAUTH2_PROVIDER_APPLICATION_MODEL = "oauth2_provider.Application"
```

Comment:
This was added to fix a migration issue related to using the oauth2_provider Application model as a foreign key.

https://github.com/jazzband/django-oauth-toolkit/issues/240